pkg/svcat/service-catalog/binding.go

```diff
@@ -264,3 +265,69 @@ func (sdk *SDK) bindingHasStatus(binding *v1beta1.ServiceBinding, status v1beta1
 
 	return false
 }
+
+// RemoveFinalizersForBinding removes all finalizers for a single binding
```

Comment:
nit:  RemoveFinalizerForBinding (singular finalizer)
here and other places:  "**all** finalizers"  but we're actually only deleting the v1beta1.FinalizerServiceCatalog finalizer (not all finalizers).  Code is correct.